lib/src/interactions.dart

```diff
@@ -66,12 +66,12 @@ abstract class IInteractions {
   Future<void> deleteGuildCommands(List<Snowflake> guildIds);
 
   /// Fetches all global bots command
-  Stream<ISlashCommand> fetchGlobalCommands();
+  Stream<ISlashCommand> fetchGlobalCommands({bool withLocales = true});
```

Comment:
`withLocales` should be removed here

Reply:
Can you explain me why? I don't get it..

Reply:
You have removed it from the HTTP endpoints so there is no need to have it here: https://github.com/nyxx-discord/nyxx_interactions/pull/48/commits/7c4572db3fb2c982e4444da1aa18c4015ffeaab9

Reply:
Yeah, but I have removed from `fetch{Global/Guild}Command` not `fetch{Global/Guild}Commands`

Reply:
Ah, I misread the diff and though you'd removed it from both. My bad!